frontend/src/pages/index.js

```diff
@@ -1,5 +1,17 @@
+import { useEffect, useState } from "react";
+
 export default function Home() {
-  return <h1>
-    Hello Team!
-  </h1>;
+  const [message, setMessage] = useState("Loading...");
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
just be careful of these changes, they persisted from another branch since you git checkout -b from this branch (https://github.com/DSC-McMaster-U/Auto-ML/tree/frontend-request) and not main